const { catchError } = require("../utils/asyncCatch");
exports.createNew = catchError(async (schema, res, req) => {
  // const data = await schema.create(re)
});
